test/message/promise_handled_reject.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+const common = require('../common');
+
+// Flags: --expose-gc
+
+var p = new Promise(function(res, rej) {
+  consol.log('oops'); // eslint-disable-line no-undef
+});
+
+// Manually call GC due to possible memory contraints with attempting to
+// trigger it "naturally".
+setTimeout(common.mustCall(() => {
```

Comment:
These are `message` tests for a reason. Adding that handler would prevent this from functioning.